extension/settings/prod.js

```diff
@@ -2,5 +2,7 @@
 const settings = {
     logTelemetry: false,
     uploadTelemetry: true,
-    sendDebugPing: false
+    sendDebugPing: false,
+    sentryDsn: "https://9c324392ba52495d8e2de947420950a5@o1069899.ingest.sentry.io/6293991",
```

Comment:
just a layman's question: what's the mechanism used by sentry to prevent spam here?

Reply:
https://forum.sentry.io/t/how-does-sentry-prevent-spammers/8188 there is no protection on Sentry side, but I set a limit of 30 events per minute and we can also configure server-side filtering of events if we see something weird there